src/db/util.py

```diff
@@ -24,12 +24,16 @@ def create_connection_cursor(db_uri_str=None, encapsulate_exception=True):
     return connection, cursor
```

Comment:
I suggest to use PG_URI_STR instead of PG_CONN in create_connection_cursor method

Reply:
Then we can also remove `db.PG_CONN = PG_CONN` from layman settings, and also `PG_CONN` from `src/db/__init__.py`